### PR TITLE
git-octopus 1.0 (new formula)

### PR DESCRIPTION
**Pull request by mathieubolla**
Hi,

I've created a formula for git-octopus, a tool developed at [lesfurets.com](https://www.lesfurets.com) that we open sourced. This tools enables one to check his current git branch will merge on master along with all other branches his colleagues are working on. This is something used at scale at lesfurets.com by 5 teams totaling 25 developers.

This is the initial revision of the formula, following the first stable release of the tool recently.

Thank you,

Mathieu

### CODE REVIEW

File: Library/Formula/git-octopus.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class GitOctopus < Formula
+  homepage "https://github.com/lesfurets/git-octopus"
+  url "https://github.com/lesfurets/git-octopus/archive/v1.0.tar.gz"
```
Comment (DomT4) [inline Library/Formula/git-octopus.rb]: Need a ` sha256 ` line here as well. Non-checksummed formulae won't be accepted.

File: Library/Formula/git-octopus.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  def install
+    bin.install "bin/git-octopus"
+    man1.install "man/man1/git-octopus.1"
+  end
+  test do
```
Comment (DomT4) [inline Library/Formula/git-octopus.rb]: As much as Homebrew likes compact formulae, please don't squash the blocks together quite *this* much - Makes it incredibly difficult to read through, heh. An empty line between each major block is fine.

**Comment by mathieubolla** [comment]
I've taken your comments into account, and will submit again. Now closing this one. Thank you.

**Comment by DomT4** [comment]
Thanks! Feel free to amend the existing PR rather than opening a new one if that's easier - Whichever you prefer.

**Pull request closed without merging**